react-ui/src/components/Layouts/InputWorkerTimes/TimeSlot.js

```diff
@@ -0,0 +1,84 @@
+import React  from 'react';
+import { render } from 'react-dom';
+import './TimeSlot.css'; 
+import {moment} from 'moment';
+
+
+/* import {Container, Row, Col} from 'react-grid-system'; */
+
+class TimeSlot extends React.Component { // this is a class component for the checkbox feature of selecting recurring days
+  constructor() {
+    super();
+    this.state = {
+      Times: [
+        {id: 1, value: "9:00", endtime:"9:30"},
```

Comment:
Hi Kyle,
I would suggest moving this into a constants file in case other functions need it in the future (so unnecessary repetition is avoided). I assume that this is just a temporary measure until the endpoint is made to fetch all the timeslots so another alternative is to write a brief comment saying that this is temporary and should be replaced with that endpoint once completed.

Reply:
assuming that Melissa has worked on S4-T6, this code will not be needed, but for now, i put a comment stating this is a temporary measure

Reply:
code has been deleted for now, if needed, I can retrieve it from a previous branch